concordium-contracts-common/src/types.rs

```diff
@@ -627,6 +661,18 @@ impl fmt::Display for Duration {
 #[cfg_attr(feature = "fuzz", derive(Arbitrary))]
 pub struct AccountAddress(pub [u8; ACCOUNT_ADDRESS_SIZE]);
 
+#[cfg(feature = "concordium-quickcheck")]
+impl quickcheck::Arbitrary for AccountAddress {
+    fn arbitrary(g: &mut Gen) -> AccountAddress {
+        //todo should it satisfy any properties, or are random bytes okay?
+        let mut bytes = [0u8; ACCOUNT_ADDRESS_SIZE];
+        for byte in bytes.iter_mut() {
+            *byte = quickcheck::Arbitrary::arbitrary(g);
+        }
```

Comment:
Doesn't quickcheck have an arbitrary instance for arrays?

Reply:
I don't think so, I looked at the source code, but couldn't find any.